enterprise_access/apps/api/v1/tests/test_subsidy_access_policy_views.py

```diff
@@ -1719,13 +1723,85 @@ def test_credits_available_endpoint_with_content_assignments(
             'spend_limit': 1000000,
             'late_redemption_allowed_until': None,
             'per_learner_enrollment_limit': None,
+            'learner_requests': [],
             'per_learner_spend_limit': None,
             'assignment_configuration': str(assignment_configuration.uuid),
             'learner_credit_request_config': str(learner_credit_config.uuid),
         }
         self.assertEqual(response_json[0]['learner_content_assignments'][0], expected_learner_content_assignment)
         self.assertEqual(response_json[0], expected_response)
 
+    def test_can_request_endpoint(self):
+        """
+        Test the can-request endpoint for subsidy access policy.
+        """
+        # Create a policy with BnR enabled
+        learner_credit_config = LearnerCreditRequestConfiguration.objects.create()
+        policy = AssignedLearnerCreditAccessPolicyFactory(
+            display_name="Test BnR Policy",
+            enterprise_customer_uuid=self.enterprise_uuid,
+            active=True,
+            spend_limit=1000000,
+            subsidy_uuid=uuid4(),
+            catalog_uuid=uuid4(),
+            learner_credit_request_config=learner_credit_config,
+        )
```

Comment:
For MVP, we are only focusing on enabling browse and request on `PerLearnerSpendCreditAccessPolicy`.